panther_analysis_tool/main.py

```diff
@@ -159,9 +171,10 @@ def zip_analysis(args: argparse.Namespace) -> Tuple[int, str]:
         # Always zip the helpers
         analysis = []
         files: Set[str] = set()
-        for (file_name, f_path, spec) in list(load_analysis_specs(
-                args.path)) + list(load_analysis_specs(HELPERS_LOCATION)):
-            if file_name not in files:
+        for (file_name, f_path, spec,
+             error) in list(load_analysis_specs(args.path)) + list(
+                 load_analysis_specs(HELPERS_LOCATION)):
+            if file_name not in files and not error:
```

Comment:
It should be extremely unlikely for this to ever return an error, because if the running `test_analysis` encounters any errors on lines 159-162 we exit before getting to this point correct? Are there specific cases you encountered or you're anticipating with this error check?

Reply:
That's a good point. I hadn't had a particular case in mind, I can remove it since its a duplicate error check